include/pybind11/detail/common.h

```diff
@@ -1344,5 +1344,8 @@ constexpr
 #    define PYBIND11_BACKWARD_COMPATIBILITY_TP_DICTOFFSET
 #endif
 
+// TODO: determine which platforms cannot use thread_local.
```

Comment:
I thought it was one of the iOS platforms.  But that should've failed one of the PR Checks and didn't.  Meaning all of the Checks pybind11 currently has support `thread_local` anyway.... can that be right?

Reply:
I think we might have fixed the issue with iOS, you have to set some minimum version to be able to use it and it wasn't letting us set the minimum version. But now I think it's working.